### DISCUSSION TITLE
How to work the USB Camera?

### DISCUSSION THREAD

**Discussion by VanTrook** [FIRST-Tech-Challenge/FtcRobotController#366] [Q&A] [answered]
I have spent a very long time trying to make the USB camera work on the driver hub, to no avail. Does anybody even know where to begin on the camera? It's already configured to the driver hub.

**Comment by texasdiaz** [comment, marked as answer]
Greetings! The USB Webcam is not designed to work on the Driver Hub or even within the [Driver Station](http://ftc-docs.firstinspires.org/control_hard_compon/ds_components/index.html) in general. The USB Webcam is intended to only be used on the robot. If you're using a [Control Hub](http://ftc-docs.firstinspires.org/control_hard_compon/rc_components/index.html) on the robot, the USB webcam can plug directly into any one of the Control Hub USB-A ports. However, if you're using an [Android Smartphone and a REV Expansion Hub](http://ftc-docs.firstinspires.org/control_hard_compon/rc_components/index.html), you need a slightly different configuration that will require a Powered USB Hub. In the basic configuration, you would use the rear camera of the Android Phone as the vision camera.
